api/types/container/host_config.go

```diff
@@ -313,7 +313,7 @@ type HostConfig struct {
 	Runtime         string            `json:",omitempty"` // Runtime to use with this container
 
 	// Applicable to Windows
-	ConsoleSize [2]int    // Initial console size
+	ConsoleSize [2]uint   // Initial console size
```

Comment:
@mlaventure Updated to use type `Box` as per this comment in the matching OCI spec PR (https://github.com/opencontainers/runtime-spec/pull/563#r79296389). Also updated it to `ConsoleSize` as per https://github.com/opencontainers/runtime-spec/pull/563#issuecomment-247830402, dropping `Initial`. I'll open a follow-up PR to update the HCSShim APIs to also use a structure and re-vendor with the changes once this one is in.